### ISSUE TITLE
Jackett Fails to launch

### ISSUE THREAD

**Issue by shiboman** [Jackett/Jackett#9980]
*Jackett directory*: /Applications/Jackett
*launchctl info*:
org.user.Jackett = {
	active count = 0
	path = /Users/shiboman/Library/LaunchAgents/org.user.Jackett.plist
	state = spawning

	program = /Applications/Jackett/jackett
	arguments = {
		/Applications/Jackett/jackett
		--NoRestart
	}

	working directory = /Applications/Jackett

	inherited environment = {
		Apple_PubSub_Socket_Render => /private/tmp/com.apple.launchd.62i3L7Lbtt/Render
		SSH_AUTH_SOCK => /private/tmp/com.apple.launchd.zGnT2OhdDn/Listeners
	}

	default environment = {
		PATH => /usr/bin:/bin:/usr/sbin:/sbin
	}

	environment = {
		PATH => /usr/bin:/bin:/usr/sbin:/sbin
		XPC_SERVICE_NAME => org.user.Jackett
	}

	domain = com.apple.xpc.launchd.user.domain.501.100007.Aqua
	asid = 100007
	minimum runtime = 10
	exit timeout = 5
	runs = 1
	successive crashes = 0
	excessive crashing = 0
	last exit code = 130

	event triggers = {
	}

	endpoints = {
	}

	dynamic endpoints = {
	}

	pid-local endpoints = {
	}

	instance-specific endpoints = {
	}

	event channels = {
	}

	sockets = {
	}

	spawn type = daemon
	spawn role = (null)
	jetsam priority = 3
	jetsam memory limit (active) = (unlimited)
	jetsam memory limit (inactive) = (unlimited)
	jetsamproperties category = daemon
	cpumon = default

	properties = {
		partial import = 0
		launchd bundle = 0
		xpc bundle = 0
		keepalive = 1
		runatload = 1
		dirty at shutdown = 0
		low priority i/o = 0
		low priority background i/o = 0
		legacy timer behavior = 0
		exception handler = 0
		multiple instances = 0
		supports transactions = 0
		supports pressured exit = 0
		enter kdp before kill = 0
		wait for debugger = 0
		app = 0
		system app = 0
		creates session = 0
		inetd-compatible = 0
		inetd listener = 0
		abandon process group = 0
		one-shot = 0
		event monitor = 0
		penalty box = 0
		pended non-demand spawn = 0
		role account = 0
		launch only once = 0
		system support = 0
		app-like = 0
		inferred program = 1
		joins gui session = 0
		joins host session = 0
		parameterized sandbox = 0
		resolve program = 0
		abandon coalition = 0
		high bits aslr = 0
		extension = 0
		nano allocator = 0
		no initgroups = 0
		start on fs mount = 0
		endpoints initialized = 1
		disallow all lookups = 0
		system service = 0
	}
}
*LaunchAgents permissions*:
drwxr-xr-x@  5 shiboman  staff   160 Oct 20 19:42 .
drwx------@ 61 shiboman  staff  1952 Oct 25 12:07 ..
-rw-r--r--   1 shiboman  staff   684 Oct 26 15:59 org.user.Jackett.plist

**Comment by bsplosion** [comment]
Same issue here, Windows 10 though.  No logs that I can find, though it looks like executables in ProgramData were updated at 4PM EST today.

**Comment by ilike2burnthing** [comment]
Install and uninstall scripts have been updated by https://github.com/Jackett/Jackett/pull/13626, available in [v0.20.2080](https://github.com/Jackett/Jackett/releases/tag/v0.20.2080). Please check if your issue is resolved.
